client/blocks/upe/index.js

```diff
@@ -59,25 +58,19 @@ Object.entries( paymentMethodsConfig )
 		registerPaymentMethod( upeElement( method, api, config ) );
 	} );
 
-if ( getBlocksConfiguration()?.isECEEnabled ) {
-	// Register Express Checkout Elements.
-	if (
-		getBlocksConfiguration()?.isAmazonPayAvailable && // Hide behind feature flag so the editor does not show the button.
-		getBlocksConfiguration()?.isAmazonPayEnabled
-	) {
-		registerExpressPaymentMethod( expressCheckoutElementAmazonPay( api ) );
-	}
-	if ( getBlocksConfiguration()?.isPaymentRequestEnabled ) {
-		registerExpressPaymentMethod( expressCheckoutElementApplePay( api ) );
-		registerExpressPaymentMethod( expressCheckoutElementGooglePay( api ) );
-	}
-	if ( getBlocksConfiguration()?.isLinkEnabled ) {
-		registerExpressPaymentMethod( expressCheckoutElementStripeLink( api ) );
-	}
-} else {
-	// Register Stripe Payment Request.
-	// TODO: We can remove this once we're sure everyone on the new checkout (UPE) has been migrated to ECE.
```

Comment:
Related to my previous comment about upgrades: is it possible to have UPE + payment request buttons?